client/pages/addons/recent/recent.js

```diff
@@ -1,15 +1,26 @@
-import React from 'react'
+import React, { Component } from 'react'
+import { PropTypes as MobxPropTypes } from 'mobx-react'
 
+import AddonCollection from '../../../components/addons/addon-collection'
 import AddonsNav from '../addons-nav'
 
-const RecentAddonsPage = () => (
-	<div>
-		<AddonsNav />
+class RecentAddonsPage extends Component {
+	static propTypes = {
+		addons: MobxPropTypes.observableArrayOf(MobxPropTypes.observableObject).isRequired
```

Comment:
Don't think this is being used? But it probably should be, and passed into `<AddonCollection>` as a prop as I said above.